kustomize/common/oidc-authservice/base/statefulset.yaml

```diff
@@ -15,6 +15,7 @@ spec:
       labels:
         app: authservice
     spec:
+      serviceAccountName: authservice
```

Comment:
If we aren't changing this / keeping the default from upstream do we need this addition? 
It will still exist in the manifest / `aaw.yaml` even if you remove it. 

otherwise lgtm